src/sqlancer/yugabyte/ysql/gen/YSQLCommon.java

```diff
@@ -159,8 +159,14 @@ public static void generateWith(StringBuilder sb, YSQLGlobalState globalState, E
             }
         } else if (Randomly.getBoolean()) {
             errors.add("Cannot use TABLEGROUP with TEMP table");
```

Comment:
Is this error caused by the same reason? Can we remove this error?

Reply:
I think it is not the same error. 

What we are tring to fix is: `ERROR: cannot use tablegroups in a colocated database`. see: https://github.com/sqlancer/sqlancer/issues/789

But here it is `Cannot use TABLEGROUP with TEMP table`